### PR TITLE
Add `perfect-numbers`

### PR DESCRIPTION
**Pull request by BNAndras**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -334,6 +334,14 @@
           "math"
         ]
       },
+      {
+        "slug": "perfect-numbers",
+        "name": "Perfect Numbers",
+        "uuid": "696b5088-ec86-4d60-8ac1-2cc1d89dee52",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```
Comment (glennj) [inline config.json:343]: maybe more than 2

**Review by glennj — APPROVED** [review]

**Comment by glennj** [comment]
I left this unmerged for your consideration: do you have the power to merge in this repo?

**Comment by BNAndras** [comment]
No, this rep is `maintained` so I don’t have write access as a cross-track reviewer.

File: config.json (hunk 1 above)
Comment (BNAndras) [reply config.json:343]: 2 is my default for an easy exercise but I’d be glad to tweak it. Did you have something in mind? 3 to mark it as a harder easy problem or 4 as a easier medium problem?

Comment (glennj) [reply config.json:343]: Requiring an enum bumps it up for me. I hate quibbling over gradations of "easy", particularly when it's completely invisible on the site. But it's not a medium difficulty exercise.

Leave it as it then, not worth changing.

**Pull request merged**